source/_posts/2015-09-03-hiring-apprentices.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Hiring Apprentices"
+date: 2015-09-09 14:44:39 -0600
+comments: true
+categories: hiring, apprenticeship, mentoring, ruby
+---
+
+The [MX](http://mx.com/) apprenticeship has been a lot of work and fun so far.
+This blog post is the beginning of an effort to document what I've learned along the way.
+
+# The Goal
+
+Initially I got interested in the idea of an apprenticeship while listening to the [ruby rogues episode with Joe Mastey and Jill Lynch](https://devchat.tv/ruby-rogues/190-rr-apprenticeship-with-joseph-mastey-and-jill-lynch-of-enova).
+I help to organize a [local ruby meetup group](http://utruby.org/#uv.rb) so I liked the idea of helping new engineers.
+I also love my job at MX so I liked the idea of helping to grow our team.
+
+When I discussed the idea of an apprenticeship with people at MX, the first thing I had to do was nail down what our objective was. "Helping new engineers" wasn't going to cut it.
+
+We decided that the primary objective was to __add amazing engineers to the team__.
+
+# Amazing Apprentices
+
+Based on my experience in the local Ruby community I believed that there were some really amazing programmers who were self-taught or graduating from bootcamps that were struggling to find first jobs.
```

Comment:
Comma after "community." It's an introductory phrase.

Take out "there." It should read "...I believed there were some really..."

Use "who" instead of "that" because people. Alternatively, you can also use an "and." Also, add a "their." Like so:  "...bootcamps who/and were struggling to find THEIR first jobs."